desmume/src/frontend/posix/cli/main.cpp

```diff
@@ -314,15 +314,15 @@ static void desmume_cycle(struct ctrls_event_config * cfg)
         (SDL_PollEvent(&event) || (!cfg->focused && SDL_WaitEvent(&event))))
       {
         process_ctrls_event( event, cfg);
-    }
+      }
```

Comment:
please dont gratuitously change whitespace, even if it appears wrong this should be done as a separate cleanup PR.